tools/gc_decode.cpp

```diff
@@ -24,15 +25,18 @@ void str_to_uint16(char *str, uint16_t *res, uint8_t base) {
 }
 
 void usage_error(char *name) {
-  std::cerr << "Usage: " << name << " [-raw] <global_code>" << std::endl
+  std::cerr << "Usage: " << name << " [-gc] <global_code>" << std::endl
```

Comment:
The `-raw` argument is used to dump the result out in raw format. You have overloaded this argument.
Either change the existing argument to something like `-dump` or `-rawout` if you want to use `-raw` to indicate a raw format as input.